android/modules/android/src/java/ti/modules/titanium/android/AndroidModule.java

```diff
@@ -325,6 +326,11 @@ public RProxy getR() {
 		return r;
 	}
 
+	@Kroll.method @Kroll.getProperty
+	public ActivityProxy getCurrentActivity() {
+		return TiApplication.getAppCurrentActivity().getActivityProxy();
```

Comment:
It is possible for `getAppCurrentActivity()` to return `null`, best add a guard.